components/support/ktx/src/test/java/mozilla/components/support/ktx/android/content/res/Resources.kt

```diff
@@ -0,0 +1,25 @@
+package mozilla.components.support.ktx.android.content.res
+
+import android.util.DisplayMetrics
+import org.junit.Assert
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.RuntimeEnvironment
+
+@RunWith(RobolectricTestRunner::class)
+class ResourcesTest {
```

Comment:
@SubhrajyotiSen I think the filename should be ResourcesTest.kt as well for consistency. Thanks!